### PR TITLE
fix(route): blockbeats

### LINKED ISSUE #11771: blockbeats RSS订阅返回错误
### 路由地址

```routes
/blockbeats/:channel?
```

### 完整路由地址

```fullroutes
/blockbeats/news
```

### 相关文档

https://docs.rsshub.app/new-media.html#lu-dong

### 预期是什么？

订阅成功

### 实际发生了什么？

订阅失败

### 部署

RSSHub 演示 (https://rsshub.app)

### 部署相关信息

_No response_

### 额外信息

```shell
Looks like something went wrong
Route requested: /blockbeats/news
Error message: this route is empty, please check the original site or create an issue
Helpful Information to provide when opening issue:
Path: /blockbeats/news
Node version: v16.19.0
Git Hash: 6bed7cb
```

### 这不是重复的 issue

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保该错误尚未被报告。

### PR DESCRIPTION
**Pull request by jameshih**
## 该 PR 相关 Issue / Involved issue

Close #11771

## 完整路由地址 / Example for the proposed route(s)

```routes
/blockbeats/newsflash
/blockbeats/article
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [ ] 新的路由 New Route
  - [ ] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [ ] 文档说明 Documentation
  - [ ] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

**Comment by TonyRL** [comment]
Please remove any changes on yarn.lock unless you add a new package

**Review by TonyRL — CHANGES_REQUESTED** [review]
Those feeds are still returning empty items

**Review by TonyRL — COMMENTED** [review]
You can use API endpoints instead of looking at the HTML
https://api.theblockbeats.info/v3/newsflash/select
https://api.theblockbeats.info/v3/Information/newsall

File: lib/v2/blockbeats/index.js [hunk 1]
```diff
@@ -1,50 +1,49 @@
...
+            const description = channel === 'newsflash' ? $(item).find('div.news-flash-item-content').html() : $(item).find('div.home-news-lft-content').text();
             return {
                 title,
                 link,
-                pubDate,
+                description,
             };
         })
-        .filter((_, item) => item.title && item.link.includes(channel))
         .get();
```
[the remark this answers is not in the document]
Comment (TonyRL) [reply lib/v2/blockbeats/index.js:27]: This needs to be resolved before merging.

[the remark this answers is not in the document]
Comment (jameshih) [reply lib/v2/blockbeats/index.js:27]: Working on it

**Pull request merged**